packages/teams-js/src/internal/messageObjects.ts

```diff
@@ -1,22 +1,61 @@
+import { UUID as MessageUUID } from './interfaces';
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * MessageIDs represent the legacy number id used for processing MessageRequests and MessageResponses
+ */
+export type MessageID = number;
+
 /**
  * @internal
  * Limited to Microsoft-internal use
  */
 export interface MessageRequest {
-  id?: number;
+  id?: MessageID;
+  uuid?: MessageUUID;
+  func: string;
+  timestamp?: number;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  args?: any[];
+  apiVersionTag?: string;
+  isPartialResponse?: boolean;
+}
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export interface SerializedMessageRequest {
+  id?: MessageID;
+  uuidAsString?: string;
   func: string;
   timestamp?: number;
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   args?: any[];
   apiVersionTag?: string;
 }
 
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export interface SerializedMessageResponse {
+  id: MessageID;
+  uuidAsString?: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  args?: any[];
```

Comment:
>any [](http://example.com/codeflow?start=9&length=3)

Can `unknown` be used here, instead of having to disable the any warning?

Reply:
Looked into this and this would cause issues with some unit tests, I'll make a work item to look into making this change in the future, but for now will keep it as a `any[]`

Reply:
Thank you!